tools/add_triage_signature.py

```diff
@@ -494,42 +497,36 @@ def _update_ticket(self, url, issue_key, should_update=False):
         cluster = md['cluster']
         cluster_id = cluster['id']
 
-        with TemporaryDirectory() as tempdir:
-            cluster_logs_file = os.path.join(tempdir, "cluster_logs.tar")
-            try:
-                get_cluster_logs(url, cluster_id, cluster_logs_file)
-            except Exception as e:
-                logger.error("Error getting logs for %s at %s: %s, they may have been deleted", issue_key, url, e)
-                return
-
-            with tarfile.open(cluster_logs_file) as tar:
-                tar.extractall(tempdir)
-
-            hosts = []
-
-            for tar_file in os.listdir(tempdir):
-                tar_file = os.path.join(tempdir, tar_file)
-                try:
-                    dmesg_file = extract_file_from_tar(tar_file, file_substring="dmesg", output_dir=tempdir)
-                except FileNotFoundError:
-                    continue
-
-                host_id = get_host_id_for_host_tar(tar_file)
-                host = list(filter(lambda host: host['id'] == host_id, cluster['hosts']))[0]
+        hosts = []
 
-                logs = search_patterns_in_file(dmesg_file, self.ERRORS_PATTERNS)
-                logs = [(group[0], len(group)) for group in group_similar_strings(logs, ratio=80)]
+        for host in cluster["hosts"]:
+            host_id = host["id"]
 
-                for log in logs:
-                    hosts.append(OrderedDict(
-                        id=host['id'],
-                        hostname=self._get_hostname(host),
-                        log=log[0],
-                        similar_logs=log[1]))
+            try:
+                dmesg_file = get_host_log_file(log_cache=self.log_cache,
+                    cluster_url=url, host_id=host_id, cluster_id=cluster_id,
+                    filename="dmesg.logs")
+            except FileNotFoundError:
+                # Host doesn't have dmesg
+                continue
+            except requests.exceptions.HTTPError as e:
+                if e.errno == 404:
+                    logger.warning(f"{browse(issue_key)} doesn't have a log tar, skipping")
+                continue
```

Comment:
maybe we should at least log the error?

Reply:
Forgot to raise in case the error is not 404. But on 404 I intentionally don't log the error because it's really spammy